.github/workflows/download_trivy_db.yaml

```diff
@@ -0,0 +1,40 @@
+name: Update Trivy Cache
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # Update daily at 00:00 - before scheduled trivy scan
+  workflow_dispatch:  # Allow manual triggering
```

Comment:
if you want to make it reusable you should add 
```yaml
on:
  workflow_call:
    inputs: [...] # not sure if these are needed
```

Reply:
🤔 If we make this reusable, we we'll also need to open 20+ PRs for all the repositories using them, right? I guess we cannot have a cross-repo cache...

Reply:
I guess these PRs need to be open as well to reference this one, yes 🫠

Reply:
Added in https://github.com/newrelic/coreint-automation/pull/84/commits/1f03e76f865749e8f92eee74522f9831ef0b6690